src/webui/webapplication.cpp

```diff
@@ -234,13 +226,13 @@ void WebApplication::action_query_torrents()
     CHECK_URI(0);
     const RawStringMap& gets = request().gets;
 
-    print(btjson::getTorrents( toUtf8(gets["filter"]),
-                               toUtf8(gets["category"]),
-                               toUtf8(gets["sort"]),
-                               toUtf8(gets["reverse"]) == "true",
-                               toUtf8(gets["limit"]).toInt(),
-                               toUtf8(gets["offset"]).toInt()),
-           Http::CONTENT_TYPE_JSON);
+    print(btjson::getTorrents(gets["filter"],
+                              QString::fromUtf8(gets["category"].constData(), gets["category"].length()),
```

Comment:
As far as I know, a general Utf-8 byte sequence could contain null byte somewhere in the middle. So I decided to be a little cautious by supplying length. Certainly 
a category could never contain contain such a symbol.
I will remove the length.